core/src/main/java/com/linecorp/armeria/common/metric/MoreMeters.java

```diff
@@ -38,9 +41,60 @@
 
     private static final double[] PERCENTILES = { 0, 0.5, 0.75, 0.9, 0.95, 0.98, 0.99, 0.999, 1.0 };
 
+    private static volatile DistributionStatisticConfig distributionStatisticConfig =
+            DistributionStatisticConfig.builder()
+                                       .percentiles(PERCENTILES)
+                                       .build();
+
+    /**
+     * Sets {@link DistributionStatisticConfig} to {@link MoreMeters}.
+     * Any {@link Timer} and {@link DistributionSummary} which created by {@link MoreMeters}
+     * using {@link MoreMeters#newTimer(MeterRegistry, String, Iterable)} and
+     * {@link MoreMeters#newDistributionSummary(MeterRegistry, String, Iterable)} will use this config.
+     */
+    public static void setDistributionStatisticConfig(DistributionStatisticConfig config) {
+        requireNonNull(config, "distributionStatisticConfig");
+        distributionStatisticConfig = config;
+    }
+
+    /**
+     * Returns the current {@link DistributionStatisticConfig} of {@link MoreMeters}.
+     */
+    public static DistributionStatisticConfig distributionStatisticConfig() {
+        return distributionStatisticConfig;
+    }
+
+    /**
+     * Returns a newly-registered {@link DistributionSummary} configured by
+     * {@link MoreMeters#distributionStatisticConfig}.
+     **/
+    public static DistributionSummary newDistributionSummary(MeterRegistry registry,
+                                                             String name, Iterable<Tag> tags) {
+        requireNonNull(registry, "registry");
+        requireNonNull(name, "name");
+        requireNonNull(tags, "tags");
+
+        return DistributionSummary.builder(name)
+                                  .tags(tags)
+                                  .publishPercentiles(distributionStatisticConfig.getPercentiles())
+                                  .publishPercentileHistogram(
+                                          distributionStatisticConfig.isPercentileHistogram())
```

Comment:
All setter calls could be joined into a single line:

```java
.publishPercentileHistogram(distributionStatisticConfig.isPercentileHistogram())
.maximumExpectedValue(distributionStatisticConfig.getMaximumExpectedValue())
...
```

Reply:
this line is over checkStyle quota when joins

Reply:
That's why I suggested renaming the field to something like `distStatCfg`.

Reply:
Would you change the name plz?

Reply:
fixed